encoder/libav_encoder.cpp

```diff
@@ -65,7 +65,20 @@ const std::map<std::string, std::function<void(VideoOptions const *, AVCodecCont
 
 void LibAvEncoder::Signal()
 {
-	feed_encoder_frames_ = !feed_encoder_frames_;
+	// We now need to tell the rest of the program what is happening
+	pausestate_ = !pausestate_;
```

Comment:
Our convention in libcamera_apps is normally to put an underscore in variable names to break words, so it would be `paused_state`. But see my other comment below!